CHANGELOG.adoc

```diff
@@ -16,7 +16,7 @@ For a detailed view of what has changed, refer to the {url-repo}/commits/main[co
 === Breaking changes
 
 Improvement::
-
+* Add `setContext` function to StructuralNode. Move `getContext`
```

Comment:
Your PR doesn't seem to be moving getContext() anymore.
Additionally, I don't think it is a breaking change.
Could you please fix this in the changelog?

Reply:
Yup.